LayoutKitSampleApp/Benchmarks/FeedItemLayoutKitView.swift

```diff
@@ -13,7 +13,13 @@ import ExampleLayouts
 class FeedItemLayoutKitView: UIView, DataBinder {
 
     private var layout: FeedItemLayout? = nil
-
+    
+    private lazy var heightConstraint: NSLayoutConstraint = {
+        let constraint = NSLayoutConstraint(item: self, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: self.bounds.height)
```

Comment:
This is a minor point, but please see point 1.10 of https://github.com/linkedin/swift-style-guide#1-code-formatting for the preferred formatting style for long lines such as this one.